src/main.c

```diff
@@ -942,6 +942,8 @@ static void print_usage(void)
 	"  -s <site>     Connect/grant to a different site\n"
 	"  -F            Try to grant the ticket immediately\n"
 	"                even if not all sites are reachable\n"
+	"                Only for manual tickets:\n"
```

Comment:
Would it be less confusing without "Only"? ;-)

Reply:
ok